### DISCUSSION TITLE
Strange box-shadows that only appear in production

### DISCUSSION THREAD

**Discussion by olikami** [vercel/next.js#11963] [Help]
I feel a bit stupid here, I made a small personal project which looks fine when I'm developing, however when I push it to production I get a bunch of strange box shadows.

This is how it looks when developing (running `npm run dev`):
![Bildschirmfoto 2020-04-16 um 23 26 42](https://user-images.githubusercontent.com/3026203/79508435-da2b8d80-8039-11ea-9a2f-76c1f626524e.png)

and this is what I get on production (through `npm run build` and `npm run start`):
![Bildschirmfoto 2020-04-16 um 23 26 49](https://user-images.githubusercontent.com/3026203/79508442-dd267e00-8039-11ea-8fc4-4ef0c89d571a.png)

The offending class here is `jss15`.

You can see the running example [here](https://tools.oli.fyi/podcast/this-american-life).

I'm happy to provide any and all additional info to debug this.

Cheers

**Comment by zackkrida** [comment]
The div with the class `MuiBox-root jss96` has a margin that is pushing your content. If you change that element's `margin: 64px` to `padding: 64px` you'll be good.

<img width="1443" alt="Screen Shot 2020-04-16 at 17 35 25" src="https://user-images.githubusercontent.com/6351754/79508997-ab48f380-8008-11ea-8908-4bb8babe0338.png">

<img width="1443" alt="Screen Shot 2020-04-16 at 17 37 31" src="https://user-images.githubusercontent.com/6351754/79509155-f4994300-8008-11ea-93a3-716401fd249c.png">
